forest/benchmarking/quantum_volume.py

```diff
@@ -58,9 +59,9 @@ def _naive_program_generator(qc: QuantumComputer, qubits: Sequence[int],
         prog.measure(qubit, ro[idx])
 
     # restrict compilation to chosen qubits
-    isa_dict = qc.device.get_isa().to_dict()
-    single_qs = isa_dict['1Q']
-    two_qs = isa_dict['2Q']
+    isa = qc.quantum_processor.to_compiler_isa()
+    single_qs = isa.qubits
```

Comment:
@erichulburd Can you double-check me here?

Reply:
I'm not following. Could you specify which line you're referring to? If they are strings, why does that eliminate the need to convert to `int`?

Reply:
`qubits` is still a sequence of `int`s, so it seems the conversion is still necessary. Maybe I'm missing something?

Reply:
Oops, I'm sorry, I misread line 68 and 73. `qubits` is a sequence of ints (`single_qs` is a dict with string keys).